### ISSUE TITLE
QR code at `mailadm setup-bot` is broken

### ISSUE THREAD

**Issue by missytake** [deltachat/mailadm#105]
When you run `mailadm setup-bot`, a QR code is shown, and an `OPENPGP4FPR` token which is supposed to be encoded in the QR code. This encoding doesn't work anymore, and the Delta Chat QR code scanner doesn't recognize it as a valid QR code anymore, neither do other online scanners like https://scanqr.org/. I suspect the new https://pypi.org/project/qrcode/#history release from yesterday, but let's investigate properly. try.webxdc.org still works, as a reference.

- Operating System (Linux/Mac/Windows/iOS/Android):
- Delta Chat Version:
- Expected behavior:
- Actual behavior:
- Steps to reproduce the problem:
- Screenshots:
- Logs:

**Comment by missytake** [comment]
As a workaround, you can just copy-paste the OPENPGP4FPR token into the QR code scanner of the Delta Chat desktop client for now.

**Comment by missytake** [comment]
Shit, the same seems to be the case for the user adding QR codes.

**Comment by missytake** [comment]
Version 7.4.0 of the https://pypi.org/project/qrcode/#history library has introduced a bug, resulting in all QR codes mailadm generates being unreadable. We discovered this during a live test with users, very unfortunate.

I pushed a hotfix to master yesterday, pinning the qrcode version to 7.3.1 :) so it works again.

As soon as there is a working 7.4.1 release with a fix, #106 reverts the hotfix and introduces a test for our gen_qr() function, which was untested so far.
